frontend/src/actions/repos.js

```diff
@@ -0,0 +1,24 @@
+import github from 'libs/github';
+
+export const GET_TOP_REPOS = Symbol('GET_TOP_REPOS');
+export const GET_TOP_REPOS_SUCCESS = Symbol('GET_TOP_REPOS_SUCCESS');
+
+export function getTopRepos({ lang }) {
+  return dispatch => {
+    dispatch({
+      type: GET_TOP_REPOS,
+    });
+
+    return github.getTopRepos({ lang }).then(res => {
+      dispatch(onGetTopRepo(lang, res));
+    });
+  };
+}
+
+function onGetTopRepo(lang, payload) {
+  return {
+    type: GET_TOP_REPOS_SUCCESS,
+    lang,
+    payload,
+  };
```

Comment:
What is this about?

Reply:
It's part of a boilerplate. See comment on other conversation